counsel.el

```diff
@@ -4670,6 +4670,13 @@ PREFIX is used to create the key."
                `(("Functions" ,@fns)))
       items)))
 
+(defun counsel-imenu-action-other-window (x)
+  (with-ivy-window
+    (switch-to-buffer-other-window (buffer-name))
```

Comment:
Unless there is a specific reason to use the `switch-to-buffer` family of functions, it's almost always better to use the `pop-to-buffer` family of functions.  The latter behave better with respect to various user customisations.

I'd also suggest using `(current-buffer)` instead of `(buffer-name)`.

Reply:
Thanks for the feedback!